Upstream-Status: Unsubmitted

From 62aa7e80afca84a0ce2ad5ddca3091f88a29036e Mon Sep 17 00:00:00 2001
From: Roger Luethi <roger.luethi@bisdn.de>
Date: Sat, 19 Mar 2022 16:42:16 +0100
Subject: [PATCH 2/2] as5835: rename psu_serial_numer -> psu_serial_number

For the as5835-54x, the files that show the PSU serial numbers in sysfs
are currently:

/sys/bus/i2c/devices/11-0050/psu_serial_numer
/sys/bus/i2c/devices/12-0053/psu_serial_numer

Rename them to:

/sys/bus/i2c/devices/11-0050/psu_serial_number
/sys/bus/i2c/devices/12-0053/psu_serial_number

Note: the change for the kernel driver, x86-64-accton-as5835-54x-psu.c,
      does not have any effect when applied to meta-open-network-linux/
      recipes-extended/onl. The change must be made to the copied driver
      in meta-open-network-linux/recipes-kernel, too.

Signed-off-by: Roger Luethi <roger.luethi@bisdn.de>
---
 .../as5835-54x/modules/builds/x86-64-accton-as5835-54x-psu.c  | 4 ++--
 .../builds/x86_64_accton_as5835_54x/module/src/platform_lib.c | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/packages/platforms/accton/x86-64/as5835-54x/modules/builds/x86-64-accton-as5835-54x-psu.c b/packages/platforms/accton/x86-64/as5835-54x/modules/builds/x86-64-accton-as5835-54x-psu.c
index cff3ffaf..4fa736a4 100644
--- a/packages/platforms/accton/x86-64/as5835-54x/modules/builds/x86-64-accton-as5835-54x-psu.c
+++ b/packages/platforms/accton/x86-64/as5835-54x/modules/builds/x86-64-accton-as5835-54x-psu.c
@@ -87,14 +87,14 @@ enum as5835_54x_psu_sysfs_attributes {
 static SENSOR_DEVICE_ATTR(psu_present,	S_IRUGO, show_status,	NULL, PSU_PRESENT);
 static SENSOR_DEVICE_ATTR(psu_model_name, S_IRUGO, show_string,	NULL, PSU_MODEL_NAME);
 static SENSOR_DEVICE_ATTR(psu_power_good, S_IRUGO, show_status,	NULL, PSU_POWER_GOOD);
-static SENSOR_DEVICE_ATTR(psu_serial_numer, S_IRUGO, show_string,	NULL, PSU_SERIAL_NUMBER);
+static SENSOR_DEVICE_ATTR(psu_serial_number, S_IRUGO, show_string,	NULL, PSU_SERIAL_NUMBER);
 static SENSOR_DEVICE_ATTR(psu_fan_dir, S_IRUGO, show_string,	NULL, PSU_FAN_DIR);
 
 static struct attribute *as5835_54x_psu_attributes[] = {
 	&sensor_dev_attr_psu_present.dev_attr.attr,
 	&sensor_dev_attr_psu_model_name.dev_attr.attr,
 	&sensor_dev_attr_psu_power_good.dev_attr.attr,
-	&sensor_dev_attr_psu_serial_numer.dev_attr.attr,
+	&sensor_dev_attr_psu_serial_number.dev_attr.attr,
 	&sensor_dev_attr_psu_fan_dir.dev_attr.attr,
 	NULL
 };
diff --git a/packages/platforms/accton/x86-64/as5835-54x/onlp/builds/x86_64_accton_as5835_54x/module/src/platform_lib.c b/packages/platforms/accton/x86-64/as5835-54x/onlp/builds/x86_64_accton_as5835_54x/module/src/platform_lib.c
index 4ac61c6d..da7750ff 100644
--- a/packages/platforms/accton/x86-64/as5835-54x/onlp/builds/x86_64_accton_as5835_54x/module/src/platform_lib.c
+++ b/packages/platforms/accton/x86-64/as5835-54x/onlp/builds/x86_64_accton_as5835_54x/module/src/platform_lib.c
@@ -49,7 +49,7 @@ int get_psu_serial_number(int id, char *serial, int serial_len)
     char *sn = NULL;
 
     /* Read AC serial number */
-    node = (id == PSU1_ID) ? PSU1_AC_HWMON_NODE(psu_serial_numer) : PSU2_AC_HWMON_NODE(psu_serial_numer);
+    node = (id == PSU1_ID) ? PSU1_AC_HWMON_NODE(psu_serial_number) : PSU2_AC_HWMON_NODE(psu_serial_number);
     ret = onlp_file_read_str(&sn, node);
     if (ret <= 0 || ret > PSU_SERIAL_NUMBER_LEN || sn == NULL) {
         AIM_FREE_IF_PTR(sn);
-- 
2.35.1

